### PR TITLE
add function paginate page

### PR DESCRIPTION
**Pull request by ghost**
# Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Task ID: TI-106

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by mvn-haphan-dn — COMMENTED** [review]
LGTM

File: resources/views/layouts/partials/scripts.blade.php [hunk 1]
```diff
@@ -19,3 +19,5 @@
 <script src="js/jquery.stellar.min.js"></script>
 <!-- Main -->
 <script src="js/main.js"></script>
+<!-- Main -->
+<script src="js/jquery.twbsPagination.min.js"></script>
```
Comment (at-huydq) [inline resources/views/layouts/partials/scripts.blade.php]: vì sao phải dùng jquery phân trang em ?

File: public/js/showListBook.js [hunk 2]
```diff
@@ -1,29 +1,60 @@
...
+                onPageClick: function (event, pageL) {
+                    if (current_page != pageL) {
+                        current_page = pageL;
+                        getListBooks();
+                    }
+                }
+            });
+        }
+        var books = '';
+        $.each(data.data, function (key, value) {
```
Comment (at-huydq) [inline public/js/showListBook.js]: tách chỗ này ra hàm riêng giúp a nha. hàm này chỉ xử lý tạo content book.

**Review by ghost — COMMENTED** [review]
pls check my cmt

File: resources/views/layouts/partials/scripts.blade.php (hunk 1 above)
Comment (at-haytran) [reply resources/views/layouts/partials/scripts.blade.php]: Cái đó hỗ trợ thôi anh, vẫn dùng paginate trả về để xử lý mà anh.

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**